e2e/benchmarks/local-benchmark/index.html

```diff
@@ -149,6 +149,15 @@ <h2>TensorFlow.js Model Benchmark</h2>
             tunableFlagsControllers['KEEP_INTERMEDIATE_TENSORS'].setValue(false);
           }
         }
+        if (tunableFlagsControllers['CHECK_COMPUTATION_FOR_ERRORS'] != null &&
+            urlState.has('CHECK_COMPUTATION_FOR_ERRORS')) {
+          const value = urlState.get('CHECK_COMPUTATION_FOR_ERRORS');
+          if (value === 'true') {
+            tunableFlagsControllers['CHECK_COMPUTATION_FOR_ERRORS'].setValue(true);
+          } else {
+            tunableFlagsControllers['CHECK_COMPUTATION_FOR_ERRORS'].setValue(false);
+          }
+        }
```

Comment:
Can you merge the logic for WEBGL_USE_SHAPES_UNIFORMS, KEEP_INTERMEDIATE_TENSORS and this one?

Reply:
Done